pkg/plugin/aggregation/handler.go

```diff
@@ -109,6 +151,23 @@ func (h *Handler) resultsHandler(w http.ResponseWriter, r *http.Request) {
 	r.Body.Close()
 }
 
+func (h *Handler) statusHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Sorry to point out another naming thing, but for consistency I think we should use `progress` in this file. Especially in the log messages.